dds/DdsDcpsCore.idl

```diff
@@ -447,6 +447,7 @@ module OpenDDS {
     BUILT_IN_TOPIC_TYPE
     struct ParticipantLocationBuiltinTopicData {
             BUILT_IN_TOPIC_KEY OctetGuidArray16 guid;
+            DDS::BuiltinTopicKey_t participant_key;
```

Comment:
As we discussed offline, I think we can avoid adding this.  Let me know if you need more info.